### PR TITLE
add .status

### PR DESCRIPTION
**Pull request by juliangruber**
required for https://github.com/Level/levelup/pull/342

cc @ralphtheninja

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -83,7 +83,6 @@ test('test open() extensibility', function (t) {
   t.equal(spy.getCall(0).thisValue, test, '`this` on _open() was correct')
   t.equal(spy.getCall(0).args.length, 2, 'got two arguments')
   t.deepEqual(spy.getCall(0).args[0], expectedOptions, 'got default options argument')
-  t.equal(spy.getCall(0).args[1], expectedCb, 'got expected cb argument')
```
Comment (juliangruber) [inline test.js]: had to remove those since abstract-leveldown now provices its own callback

**Comment by ralphtheninja** [comment]
Otherwise lkatm (looks kick ass to me)!

**Comment by ralphtheninja** [comment]
We might want to consider splitting up the tests as well. I suspect we will have more coming shortly.

**Comment by juliangruber** [comment]
published as 2.4.0

**Comment by ralphtheninja** [comment]
:+1:

**Pull request merged**